packages/common/package.json

```
@@ -72,5 +72,17 @@
     "@types/node": "^18.15.11",
     "tsup": "^6.7.0",
     "vitest": "0.34.6"
+  },
+  "peerDependencies": {
+    "@aws-sdk/client-kms": "^3.x",
+    "asn1.js": "^5.x"
```

surprised this semver is valid, but might be more clear as:
```suggestion
    "@aws-sdk/client-kms": "3.x",
    "asn1.js": "5.x"
```